src/main/com/aca/homework/week10/concrete/map/Map.java

```diff
@@ -0,0 +1,39 @@
+package com.aca.homework.week10.concrete.map;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Map {
+
+    private List<String> values = new ArrayList<>();
```

Comment:
please create a class named Entry or Pair and keep key and value in the same class